client/signup/steps/import-preview/index.jsx

```diff
@@ -0,0 +1,228 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import { saveSignupStep, submitSignupStep } from 'state/signup/progress/actions';
+import Button from '../../../components/button';
+import Card from 'components/card';
+import ImporterLogo from 'my-sites/importer/importer-logo';
+import StepWrapper from '../../step-wrapper';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ImportPreview extends Component {
+	constructor( props ) {
+		super( props );
+
+		const { translate } = this.props;
+		this.previewData = {
+			wordpress: {
+				supported: [
+					translate( 'Posts' ),
+					translate( 'Pages' ),
+					translate( 'Images and other media' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Finding Free Images and other Media' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).